configuration.py

```diff
@@ -6,7 +6,7 @@ class Config:
     ec2_region = ""
     ec2_amis = ['']
     ec2_keypair = ''
-    ec2_secgroups = ['']
+    ec2_secgroups = [''] #ID Not Name
```

Comment:
ID must be specified if using IAM roles (explaining your issue). Otherwise, EC2 security group names work just fine. Make this clear in this comment and in the readme.